doc/examples/plot_multiblock_local_binary_pattern.py

```diff
@@ -0,0 +1,82 @@
+"""
+===========================================================
+Multi-Block Local Binary Pattern for texture classification
+===========================================================
+
+In this example, we will see how to compute the multi-block
+local binary pattern at a specified image and how to visualize it.
+
+The features are calculated in a way similar to local binary
+patterns, except that summed up pixel values
+rather than pixel values are used.
+
+`MB-LBP` is an extension of LBP that can be computed on any
+scale in a constant time using integral image. It consists of
+`9` equal-sized rectangles. They are used to compute a feature.
+Sum of pixels' intensity values in each of them are compared
```

Comment:
For each rectangle, the sum of the pixel intensities is computed.  Comparisons of these sums to that of the central rectangle determine the feature, similarly to LBP (see <insert link to LBP gallery example>).